exercises/ecn/README.md

```diff
@@ -58,26 +58,30 @@ for `h1`, `h11`, `h2`, `h22`, respectively:
    ```bash
    mininet> xterm h1 h11 h2 h22
    ```
-3. In `h2`'s XTerm, start the server that captures packets:
+4. In `h2`'s XTerm, start the server that captures packets:
    ```bash
    ./receive.py
-   ```
-4. in `h22`'s XTerm, start the iperf UDP server:
+   ```  
+5. in `h22`'s XTerm, start the iperf UDP server:
    ```bash
    iperf -s -u
    ```
-5. In `h1`'s XTerm, send one packet per second to `h2` using send.py
+
+**Note:** Since, we are intentionally creating traffic, its important to have a synchronized and simultaneous flow of packets, which will be done via 6th and 7th step.<br> Hence, at first, type both (6th & 7th) commands together in the respective Xterm windows and later press the Enter button (h1 and then h11) immediately for both Xterm windows, to clog up the traffic effectively.
```

Comment:
I'd recommend replacing "Since, we are intentionally creating traffic," with "Since we want packets sent by `h1` to at least sometimes experience long packet queues in switch `s1`,"